src/actions.ml

```diff
@@ -1795,12 +1795,23 @@ let update_pr ~bot_info (pr_info : issue_info pull_request_info) ~gitlab_mapping
           ~pr_number:pr_info.issue.number ~base:local_base_branch
           local_head_branch)
   >>= fun ok ->
+  let label = "needs: rebase" in
+  let issue = pr_info.issue.issue in
+  GitHub_queries.get_label ~bot_info ~owner:issue.owner ~repo:issue.repo ~label >>= fun label_id ->
+  GitHub_queries.get_label ~bot_info ~owner:issue.owner ~repo:issue.repo ~label:"stale" >>= fun stale_id ->
   if ok then (
```

Comment:
Note that only the query for the `needs: rebase` label really makes sense to factorize above this conditional. The one for the `stale` label could be specific to the first branch.